integration_tests/integration/incidentStatement.spec.ts

```diff
@@ -115,7 +115,7 @@ context('Incident Statement', () => {
     incidentStatementPage.radioNo().check()
     incidentStatementPage.submitButton().click()
     cy.location().should(loc => {
-      expect(loc.pathname).to.eq('/place-the-prisoner-on-report')
+      expect(loc.pathname).to.eq('/place-the-prisoner-on-report/G6415GD/3456')
     })
```

Comment:
This is the prisoner number and then the adjudications number?

Reply:
yep, the draft adjudication number